application/views/admin/survey/subview/accordion/_tokens_panel.php

```diff
@@ -106,6 +161,7 @@ function alertPrivacy()
                             : $optionsOnOff
                     ]); ?>
                 </div>
+                <span id="multiResponseHint" class="hint" style="display:none;"><?php eT("Participants will be able to enter as many responses as they want, despite what Uses Left token attribute is set to."); ?></span>
```

Comment:
https://getbootstrap.com/docs/5.3/forms/overview/ 

`<div id="multiResponseHint" class="form-text hidden">We'll never share your email with anyone else.</div>`

Reply:
If we do that, we wouldn't be able to use `toggle()` inside the code.
Also, that code is a snippet taken from Global Settings

Reply:
You mean about hidden ? 

1. Did you test
2. OK for hidden but `<div class="form-text`

Reply:
> OK for hidden but <div class="form-text

I took it from Global Settings screen

Reply:
And ? 

Global settings have dev issue : must be reported as dev issue

Here you ADD new element.